pre_commit/commands/run.py

```diff
@@ -322,11 +324,11 @@ def _has_unstaged_config(config_file: str) -> bool:
     return retcode == 1
 
 
-def _run_inner(
+def run(
         config_file: str,
         store: Store,
         args: argparse.Namespace,
-        environ: MutableMapping[str, str],
+        environ: MutableMapping[str, str] = os.environ,
```

Comment:
I feel like there might be something iffy here with mutable default values/references, but if this is how pre-commit was coded before, then we don't need to bother tinkering.

Reply:
This is how pre-commit was coded before. Generally, doing something like this is a dangerous practice, but here `run` is only ever called once, so it doesn't matter much.